aikau/src/main/resources/alfresco/lists/views/layouts/css/Row.css

```diff
@@ -0,0 +1,17 @@
+.alfresco-lists-views-layouts-Row.zebra-striping:nth-child(odd) {
+   background-color: @list-zebra-color-odd;
+}
+
+.alfresco-lists-views-layouts-Row.zebra-striping:nth-child(even) {
+   background-color: @list-zebra-color-even;
+}
+
+.alfresco-lists-views-layouts-Row.zebra-striping:nth-child(odd):hover,
+.alfresco-lists-views-layouts-Row.zebra-striping:nth-child(even):hover {
```

Comment:
Don't need the second selector, I don't think. You should just be able to use `.alfresco-lists-views-layouts-Row.zebra-striping:hover`?